docs/site/Decorators.md

```diff
@@ -147,7 +147,7 @@ class MyController {
 
 ### Parameter Decorator
 
-Syntax: see [API documentation](https://github.com/strongloop/loopback-next/blob/0739ffcfe3ef50e0bfd86055c0f4e29fd6925be0/packages/openapi-v3/src/parameter-decorator.ts#L17-L29)
+Syntax: see [API documentation](https://github.com/strongloop/loopback-next/blob/0739ffcfe3ef50e0bfd86055c0f4e29fd6925be0/packages/openapi-v3/src/decorators/parameter.decorator.ts#L17-L29)
```

Comment:
This is tricky. The link contains a SHA of a specific commit. You cannot simply edit the local path without changing the SHA, because there is no `decorators` directory as of that old SHA. OTOH, I am not sure if know the SHA of the commit that will eventually make it onto the `master` branch, one that will already contain the renames?

IMO, these links should ideally point to apidocs.loopback.io - have we fixed the problem with rendering `@param` API docs yet?

Reply:
I know there's some work that's been done already, but I don't think it's been fixed yet. If I try to search for the decorator function, I only get its namespace counterpart.

I can just have it point towards master for now.